src/main/java/graphql/schema/GraphQLSchema.java

```diff
@@ -190,7 +192,7 @@ private static ImmutableMap<String, ImmutableList<String>> buildInterfacesToObje
         return map.build();
     }
 
-    public GraphQLCodeRegistry getCodeRegistry() {
+    public @Nullable GraphQLCodeRegistry getCodeRegistry() {
         return codeRegistry;
     }
```

Comment:
Really???  This seems odd

Reply:
Check out the first constructor in this class where the code registry is set to null
```
this.codeRegistry = null;
```

In the same constructor there is also an assertion that it's not null in the builder
```
 assertNotNull(builder.codeRegistry, () -> "codeRegistry can't be null");
```

I guess the first constructor is a "partial" one, but because the object is created, we have to annotate the code registry as nullable

Reply:
Perhaps that first constructor was acting more like a builder. We could also change the constructor to avoid this nullable annotation